src/me/fru1t/subedit/utils/Utils.kt

```diff
@@ -45,4 +45,13 @@ object Utils {
   }
 
   fun askYes(): Boolean = readLine()?.toLowerCase() == "y"
```

Comment:
Add a newline to separate this function from the next

Reply:
Done